src/main/java/net/dv8tion/jda/api/entities/Emote.java

```diff
@@ -132,6 +132,14 @@ default boolean hasRoles()
      */
     boolean isManaged();
 
+    /**
+     * Whether this emote is available. A Emote is unavailable when the {@link net.dv8tion.jda.api.entities.Guild.BoostTier BoostTier}
+     * of the Guild is lower than required for this Emote.
```

Comment:
> lower than required
What defines this? We need to explain what "lower than required" means so that if a developer finds that an emote is unavailable they understand how to make it available. From the current docs, I don't understand how to do that.

Reply:
I'm not quite sure how to explain this.
The way it works is, that when a emote was added while the Discord is boosted (f.e. is the 51st emote) would it become unavailable once the Discord loses enough Boost levels to no longer have the additional emotes.

I would be glad if you had a suggestion on how I could phrase the above system in a simple sentence.

Reply:
```
Whether this emote is available for use. When an emote becomes unavailable, it cannot be used in messages. An emote becomes unavailable when the boost tier of the guild drops such that the maximum allowed emotes is lower than the total amount of emotes added to the guild. If an emote is added to the guild when the boost tier allows for more than 50 (and thus the emote is 51st emote added), then if the boost tier drops to only allow a maximum of 50, the 51st emote would be marked as unavailable.
```

Often going for the most detailed explanation is the best route.

Reply:
I've added a description which I think should be good enough to explain stuff.
I changed some stuff like trying to explain that emotes below the limit of a higher tier will remain available when the tier doesn't go lower than what they require (i.e. going from 2 to 1)